deploy/charts/istio-csr/templates/certificate.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: istiod
+  namespace: istio-system
+spec:
+  duration: 24h
+  renewBefore: 12h
+  secretName: istiod-tls
+  dnsNames:
+  - istiod.istio-system.svc
```

Comment:
Would this ever change? Probably not important for now, but is it required to use `istio-system` for your istio installation?

Reply:
As far as I know it's required, and basically hardcoded